frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -55,6 +55,21 @@ if Device:setDateTime() then
     common_settings.time = {
         text = _("Time and date"),
         sub_item_table = {
+          {
+            text = _("12-hour clock"),
+            keep_menu_open = true,
+            checked_func = function()
+                return G_reader_settings:isTrue("twelve_hour_clock")
+            end,
+            callback = function()
+                local twelve_hr = G_reader_settings:isTrue("twelve_hour_clock")
```

Comment:
Sorry, the GH suggestion diff thing is a bit difficult to work with multiple lines (if it's possible at all) but of course this line should go too.
```suggestion
```

Reply:
Resolved already, spotted that one as well just then!